rpm/valgrind.spec

```diff
@@ -27,6 +27,13 @@ malloc/new/free/delete are intercepted. As a result, Valgrind can
 detect a lot of problems that are otherwise very hard to
 find/diagnose.
 
+%package extratools
```

Comment:
It would be to serve as a warning of the large dependency and therefore the reason for the separation of these tools, rather than to declare the programming language used.